CODEOWNERS

```diff
@@ -622,6 +622,7 @@
 /samples/basic/servo_motor/boards/*microbit* @jhe
 /samples/bluetooth/                       @jhedberg @Vudentz @alwa-nordic
 /samples/boards/intel_s1000_crb/          @sathishkuttan @dcpleung @nashif
+/samples/boards/intel_socfpga/intel_socfpga_agilex_socdk/shell/* @ngboonkhai
```

Comment:
I think `/samples/boards/intel_socfpga/intel_socfpga_agilex_socdk` is enough.

Reply:
I have updated to "/samples/boards/intel_socfpga/intel_socfpga_agilex_socdk/". Done force push. Thanks.